stdlib/random.mli

```diff
@@ -141,7 +141,7 @@ module State : sig
   *)
 
   val split : t -> t
-  (** Draw a fresh PRNG state from the given PRNG state.
+  (** Draw a fresh PRNG state from the given PRNG state (which is modified).
```

Comment:
I find the parenthesis here obscure. I would use a full sentence:

> (Note that this modifies the original PRNG state, just like drawing numbers.)

Reply:
OK, I have added a separate sentence in parentheses.